content/conditions/piles/main-content-before.md

```diff
@@ -0,0 +1 @@
+## Check if it’s piles
```

Comment:
What's the pattern for this? I've noticed previously we've been using "Check if you have [condition]". So _you have_ rather than _it's_. Maybe check with the content team.